scss/_reboot.scss

```diff
@@ -48,13 +48,15 @@
 // scss-docs-start reboot-body-rules
 body {
   margin: 0; // 1
-  font-family: var(--#{$variable-prefix}body-font-family);
-  @include font-size(var(--#{$variable-prefix}body-font-size));
-  font-weight: var(--#{$variable-prefix}body-font-weight);
-  line-height: var(--#{$variable-prefix}body-line-height);
-  color: var(--#{$variable-prefix}body-color);
-  text-align: var(--#{$variable-prefix}body-text-align);
-  background-color: var(--#{$variable-prefix}body-bg); // 2
+  font-family: var(--#{$variable-prefix}body-font-family, $font-family-base);
```

Comment:
It should be standard practice that we provide a default value for css variables, it will solve many problems like above one.
As this, `$font-family-base` need a default value:
```scss
$font-family-base:            var(--#{$variable-prefix}font-sans-serif, $font-family-sans-serif) !default;
$font-family-code:            var(--#{$variable-prefix}font-monospace, $font-family-monospace) !default;
```